.env

```diff
@@ -35,6 +35,7 @@ PANDAS=latest
 DASK=latest
 TURBODBC=latest
 HDFS=2.9.2
+LIBHDFS3=2.3
```

Comment:
libhdfs3 2.3 is not supported at all. Version 2.2.31 is the last one we support but I'm not sure if that still works with the current set of conda-forge dependencies / compilers. We might propose reverting the feedstock to the last "good" version that uses the libhdfs C ABI

https://github.com/conda-forge/libhdfs3-feedstock/commit/97fd9320efd5cbac0145db60c9b867582e9a748a#diff-ae18fe4e01e61a9862ceaf165003213a

Reply:
I can build that for you when you provide me with a source tarball at a semi-official location. The original one seems to be vanished.